jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -88,14 +88,19 @@ def ConvertAndCompare(self, func_jax: Callable, *args,
     # Run TF in all execution modes
     func_tf = jax2tf.convert(func_jax)
 
+    def convert_if_bfloat16(v):
+      return jax2tf.safe_convert_to_tensor(v) if hasattr(v, "dtype") else v
```

Comment:
I think that for now (until we get more experience with jax2tf) to add here the code to convert the inputs (even code that is simple but not very efficient, e.g., going through float32). Then we can remove the safe_convert_to_tensor from the API.

Reply:
Addressed